cmd/lncli/walletrpc_active.go

```diff
@@ -412,6 +412,14 @@ var listSweepsCommand = cli.Command{
 			Name:  "verbose",
 			Usage: "lookup full transaction",
 		},
+		cli.IntFlag{
+			Name: "startheight",
+			Usage: `The start height to use when fetching sweeps.
+                                If not specified (0), the result will start from
```

Comment:
nit: we normally don't use back tick strings for the flag descriptions, as those will include the tab characters used for indenting. That then looks weird on narrow terminals (where the CLI library will attempt to render the flag text according to the terminal width).